data_dir/cdc_profile_400gb.yaml

```diff
@@ -0,0 +1,80 @@
+keyspace: cdc_test
+
+keyspace_definition: |
+
+  CREATE KEYSPACE IF NOT EXISTS cdc_test WITH replication = {'class': 'SimpleStrategy', 'replication_factor': '3'}  AND durable_writes = true;
+
+table: test_table
+
+table_definition: |
+
+  CREATE TABLE cdc_test.test_table (
+    pkid text PRIMARY KEY,
+    name text,
+    number int,
+    starttime timestamp,
+    weight decimal,
+    steps float,
+    t_num tinyint,
+    bvalue blob,
+    boy boolean,
+    vname varchar,
+    vage varint,
+    nums_set set<int>,
+    names_set set<text>,
+    class_list list<text>
+  ) WITH bloom_filter_fp_chance = 0.01
+    AND cdc = {'enabled': true, 'preimage': false, 'postimage': false, 'ttl': 3600}
+    AND caching = {'keys': 'ALL', 'rows_per_partition': 'ALL'}
+    AND comment = 'Request for unit level UIs'
+    AND compaction = {'class': 'SizeTieredCompactionStrategy'}
+    AND crc_check_chance = 1.0
+    AND dclocal_read_repair_chance = 0.1
+    AND default_time_to_live = 0
+    AND gc_grace_seconds = 864000
+    AND max_index_interval = 2048
+    AND memtable_flush_period_in_ms = 0
+    AND min_index_interval = 128
+    AND read_repair_chance = 0.0
+    AND speculative_retry = '99.0PERCENTILE';
+
+columnspec:
+  - name: pkid
+    size: fixed(1000)
+    population: seq(1..30000000)
+
+  - name: bvalue
+    size: fixed(1000)
+
+  - name: vname
+    size: fixed(1000)
+
+  - name: nums_set
+    size: fixed(5)
+    population: exp(1..5)
+
+  - name: names_set
+    size: fixed(5)
+    population: exp(1..5)
+
+  - name: class_list
+    size: uniform(1..15)
```

Comment:
The size, population have different meaning for set and list.

- items number of collection
- size of item

When I used list in old cdc_profile.yaml of longevity 4h, I can see some 40M column, the workloads fail for that.

Reply:
reference:

- https://github.com/scylladb/scylla/issues/5904#issuecomment-595083868
- https://github.com/scylladb/scylla-cluster-tests/pull/1910

Reply:
will also remove list from profile